sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/SharedAccessSignature.cs

```diff
@@ -49,7 +49,7 @@ internal class SharedAccessSignature
         private const char TokenValuePairDelimiter = '&';
 
         /// <summary>The default length of time to consider a signature valid, if not otherwise specified.</summary>
-        private static readonly TimeSpan DefaultSignatureValidityDuration = TimeSpan.FromMinutes(20);
+        private static readonly TimeSpan DefaultSignatureValidityDuration = TimeSpan.FromMinutes(30);
```

Comment:
Sweet. I'll change it to 30 minutes too. Or maybe we'll use 45, like the JS team? 🤷‍♀

Reply:
It only really matters for signatures that may be intercepted and used by an untrusted party.  Normal users won't see it because the token will be extended automatically; attackers that have intercepted will have unchallenged access for XX time.   It'd probably be good for us to all be on the same value, but I don't think it's essential.